Source/Core/VideoBackends/D3D/main.cpp

```diff
@@ -113,7 +113,7 @@ void InitBackendInfo()
 				else if (modes[i].Quality)
 					samples = StringFromFormat(_trans("%d samples (quality level %d)"), modes[i].Count, modes[i].Quality);
 				else
-					samples = StringFromFormat(_trans("%d samples"), modes[i].Count);
+					samples = StringFromFormat(_trans("%d"), modes[i].Count);
```

Comment:
```Count``` is formatted as number and passed into ```_trans```. Does anyone ever intend to translate a single digit or number? This ```_trans``` call could probably be removed.